packages/node_modules/@webex/internal-plugin-devices/src/devices.js

```diff
@@ -0,0 +1,753 @@
+// Internal dependencies.
+import {WebexPlugin} from '@webex/webex-core';
+import {safeSetTimeout} from '@webex/common-timers';
+
+import FeaturesModel from './features-model';
+
+/**
+ * Devices plugin.
+ *
+ * @description
+ * This plugin is used as a prototyped replacement plugin for the previous
```

Comment:
You should probably put a TODO here to replace this description with the actual one.

Reply:
Will move these JSDoc comments around to be more clear in regards to `TODO` vs `@description`.